libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/timeline/item/event/UtdCause.kt

```diff
@@ -12,5 +12,22 @@ enum class UtdCause {
     SentBeforeWeJoined,
     VerificationViolation,
     UnsignedDevice,
-    UnknownDevice
+    UnknownDevice,
+
+    /**
+     * Expected utd because this is a device-historical message and
+     * key storage is not setup or not configured correctly.
+     */
+    HistoricalMessage,
+
+    /**
+     * The key was withheld on purpose because your device is insecure and/or the
+     * sender trust requirement settings are not met for your device.
+     */
+    WithheldUnverifiedOrInsecureDevice,
```

Comment:
Ok but for now `WithheldUnverifiedOrInsecureDevice` and `WithheldBySender` will be unused until https://github.com/matrix-org/matrix-rust-sdk/pull/4305 is merged.

Reply:
Is it worth a comment here? so that we don't wrongly think it is already supported? 
Alternative is to remove them for now and do a PR after, as you prefer